train.py

```diff
@@ -12,40 +20,36 @@
 from wavenet import WaveNet
 
 BATCH_SIZE = 1
-CHANNELS = 256
 DATA_DIRECTORY = './VCTK-Corpus'
-FILTER_WIDTH = 2
 LOGDIR = './logdir'
 NUM_STEPS = 1000
 LEARNING_RATE = 0.10
+NETWORK = './layout.json'
```

Comment:
I like `WAVENET_PARAMS` the most.
The cmd argument could be `--wavenet_params` and the file `wavenet_params.json`.
Then the Python dictionary in the script could be called `wavenet_params` for ultimate consistency.